source/docs/software/wpilib-overview/new-for-2020.rst

```diff
@@ -1,14 +1,15 @@
 New for 2020
 ============
 
-A number of improvements have been made to FRC Control System software for 2020. This article will describe and provide a brief overview of the new changes and features as well as a more complete changelog for C++\Java WPILib changes.
+A number of improvements have been made to FRC Control System software for 2020. This article will describe and provide a brief overview of the new changes and features as well as a more complete changelog for C++\\Java WPILib changes.
```

Comment:
Why not just use a front slash like everywhere else? Aka: /

Reply:
Because I forgot that's what we did everywhere else. I'll update